docs/upgrade-notes.asciidoc

```diff
@@ -50,6 +50,24 @@ For Elastic Security solution release information, refer to {security-guide}/rel
 
 
 [discrete]
+[[breaking-201004]]
```

Comment:
How did you find out these needed to be updated too?

Reply:
Because [the link](https://github.com/elastic/kibana/pull/201004/files/3af43a82c998b4d608685072811e838bec54ee19#diff-d7680a659a0b45b0ec187364d27830ef8d2463becdbd40ca05abf1f44aff0480R31) to the docs' reference is [this value](https://github.com/elastic/kibana/pull/201004/files/3af43a82c998b4d608685072811e838bec54ee19#diff-d7680a659a0b45b0ec187364d27830ef8d2463becdbd40ca05abf1f44aff0480R8). I just kept "breaking" because of consistency with the other references